README.md

```diff
@@ -44,6 +44,7 @@ Assume we have defined a component named `Post` that is located in `/js/componen
 ```js
 import React, { Component } from "react"
 import Comments from "../Comments"
+import Post from "./Post.js"
 
 class Main extends Component {
   render () {
```

Comment:
Missing component declaration in the render method. The Post component won't display unless we add it to the render method's output:
```js
       <div>
         <Post />
         <Comments />
       </div>
```